ref/Microsoft.Build/net/Microsoft.Build.cs

```diff
@@ -412,6 +412,8 @@ public partial class ProjectUsingTaskElement : Microsoft.Build.Construction.Proj
         public Microsoft.Build.Construction.ElementLocation AssemblyFileLocation { get { throw null; } }
         public string AssemblyName { get { throw null; } set { } }
         public Microsoft.Build.Construction.ElementLocation AssemblyNameLocation { get { throw null; } }
+        public string Override { get { throw null; } set { } }
+        public Microsoft.Build.Construction.ElementLocation OverrideLocation { get { throw null; } }
```

Comment:
Can you rebase on main to pick up the PublicApiAnalyzer? that'll force you to rebuild this stuff.

Reply:
Rebased!